### PR TITLE
Add the GitHub profile message

### PR DESCRIPTION
**Pull request by tas50**
This is work that Alex put together for the profile.

Signed-off-by: Tim Smith <tsmith84@gmail.com>

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Hi, fellow open source lovers 👋
+
+We’re extremely excited that you landed on our GitHub page!
+
+At Mondoo, we’re staunch proponents of open source and all the progress that has been made in our industry because of it. A hearty thank you is in order to the community, project maintainers, and contributors. Your efforts have made people’s lives easier ❤️
+
+What does Mondoo help you do 😀
```
Comment (benr) [inline profile/README.md:7]: Should include a question mark

**Review by chris-rock — APPROVED** [review]
Great work

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+🏗 Build security into the development process from the start
+🤝 Enlist the DevOps team to help secure your company
+⏳ Save time and friction during audit cycles
+🧰 Avoid tool sprawl
+
+Wait, who is Mondoo 🤔
+
+Mondoo was founded in 2020 by DevOps and security experts who previously founded InSpec, devsec.io, and OpenStack.
```
Comment (joelddiaz) [inline profile/README.md:16]: No links to InSpec/decsec/OpenStack?

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Hi, fellow open source lovers 👋
+
+We’re extremely excited that you landed on our GitHub page!
+
+At Mondoo, we’re staunch proponents of open source and all the progress that has been made in our industry because of it. A hearty thank you is in order to the community, project maintainers, and contributors. Your efforts have made people’s lives easier ❤️
```
Comment (joelddiaz) [inline profile/README.md:5]: This reads a little weird to me. It talks about all the thanks to the communities/maintainers/contributors. Then it says `your efforts`. That can be read as if the efforts are of the person who landed at this page (who may or may not have any involvement in open source), and it makes more sense to me to say something like `All these efforts have made...`

Comment (tas50) [reply profile/README.md:5]: I also have some concerns about this bit. Any ideas on a replacement?

**Pull request merged**